client/homebrew/editor/snippetbar/snippets/cardtemplate.gen.js

```diff
@@ -0,0 +1,37 @@
+const _ = require('lodash');
+
+module.exports = function(){
+	return ['<div class="card back" style="--card-width: 67mm; --card-height: 89mm;">',
+		'# Magic Item',
+		'## Wondrous Item, Very Rare',
+		'1D10 Angry DM damage',
+		'',
+		'  <img src=\'https://i.imgur.com/iNJH8j9.png\'></img>',
+		'',
+		'  <span class="caption">Would you believe that a wizard did it?</span>',
+		'  <div class="statblock">',
+		'    **Effect:** The targeted player takes 1D10 *annoying-the-DM-in-a-dramatic-moment* damage. This damage cannot be mitigated in any way.',
+		'  </div><br />',
+		'  <div class="description centered">',
+		'      *"Don\'t say it, don\'t... You said it."*',
+		'  </div>',
+		'</div>',
```

Comment:
It seems like a lot of HTML-markup for a markdown-based tool. It probably will require some trickier CSS or some changes in markdown parser to use less HTML here, so I think it is okay to leave this as-is for the initial version of this feature. But still, we should probably submit an issue to discuss and track further improvements.

At the moment, the only thing I could suggest is to use header markup (like `####` or `#####`) for caption to simplify the markup, i.e. remove the need to type `<span class="caption"></span>".

We might also apply style using [`nth-of-type`](https://developer.mozilla.org/en-US/docs/Web/CSS/:nth-of-type) selectors, i.e. the first paragraph within a card is styled like a caption, the next one is styled like a statblock and so on, but I guess it won't be a reliable solution as it is really easy to create an extra paragraph in Markdown.

Reply:
I believe `h6` isn't widely used - it might be suitable for a `caption` style; I'll add that to the stylesheet. It may be worthwhile playing with the `h4` and `h5` styles as well - I'll make the change and see what it's like to use.